install_config/aggregate_logging.adoc

```diff
@@ -133,11 +133,12 @@ endif::openshift-enterprise[]
 
 |`openshift_logging_use_ops`
 |If set to `true`, configures a second Elasticsearch cluster and Kibana for
-operations logs. Fluentd splits logs between the main cluster and a cluster reserved
-for operations logs (which consists of *_/var/log/messages_* on nodes and the logs from the
-projects *default*, *openshift*, and *openshift-infra*). This means a second Elasticsearch
-and Kibana are deployed. The deployments are distinguishable by the *-ops* suffix included
-in their names and have parallel deployment options listed below.
+operations logs. Fluentd splits logs between the main cluster and a cluster
+reserved for operations logs, which consists of the logs from the projects
+*default*, *openshift*, and *openshift-infra*, as well as Docker, OpenShift, and
+system logs from the journal. This means a second Elasticsearch and Kibana are
```

Comment:
Do we want to say "a second Elasticsearch and Kibana instance/pod/service/etc" ? Is it one instance for both E and K or one each? Or, will users understand?